src/eval/Eval.ml

```diff
@@ -113,6 +113,9 @@ let eval_gas_charge env g =
             let%bind l' = Datatypes.scilla_list_to_ocaml l in
             pure @@ List.length l'
         | _ -> fail0 "eval_gas_charge: Can only take length of Maps and Lists" )
+    | GasCharge.MapSortCost vstr ->
+        let%bind m = Env.lookup env (mk_loc_id vstr) in
+        pure @@ EvalGas.map_sort_cost m
     | _ -> fail0 "eval_gas_charge: Must be handled by GasCharge"
```

Comment:
Do you mind adding the missing GasCharge constructors explicitly in this pattern match? I don't like the idea that we add gas charge constructors and the OCaml compiler not complaining about a missing case.